content/token22-metadata.md

```diff
@@ -0,0 +1,1083 @@
+---
+title: Metadata and Metadata Pointer Extension
+objectives:
+- Explain how the metadata pointers and metadata extensions work on Token Extension Program Mints
+- Create an NFT with metadata embedded in the mint account itself
+- Create an NFT with the metadata pointer extension
+---
+
+# Summary
+- The `metadata pointer` extension associates a public key directly to a metadata account, whether it's stored internally or via an external metadata account like Metaplex. This setup simplifies the association of tokens with their metadata.
```

Comment:
Reading this description, Metadata pointer is still unclear. "Associates a a public key" - the public key of a mint? It might be clearer to say "associates a token mint address" if this is the case. 

"extension associates a public key directly to a metadata account, whether it's stored internally" what 'it' means is not clear. If 'it' means a metadata account (as the sentence seems to indicate), I don't understand what a metadata account stored internally means - if it's a separate account, it wouldn't be internal.